getting-started/02-new-cif-project.md

```diff
@@ -0,0 +1,394 @@
+    | --------------------- | -------              | ---------------------------------- |
+    | groupId               | **com.acme.cif**     | Base Maven groupId                 |
+    | appsFolderName        | **acme**             | `/apps` folder name                |
+    | artifactId            | **acme-store**       | Base Maven ArtifactId              |
+    | version               | **1.0-SNAPSHOT**     | Version                            |
+    | package               | **com.acme.cif**     | Java Source Package                |
+    | artifactName          | **Acme Store**       | Maven Project Name                 |
+    | componentGroupName    | **Acme.Content**     | AEM component group name           |
+    | confFolderName        | **acme**             | `/conf` folder name                |
+    | contentFolderName     | **acme**             | `/content` folder name             |
+    | optionAemVersion      | *6.5.0*              | Target AEM version                 |
+    | optionIncludeExamples | *y*                  | Include Component Library examples |
+    | packageGroup          | **acme**             | Content Package Group name         |
+    | siteName              | **Acme Store**       | AEM site name                      |
+
+    > Note: If the archetype is executed in interactive mode the first time properties with default values can't be changed (see [ARCHETYPE-308](https://issues.apache.org/jira/browse/ARCHETYPE-308) for more details). The value can be changed when the property confirmation at the end is denied and the questionnaire gets repeated or by passing the parameter in the command line (e.g. `-DoptionIncludeExamples=n`).
+
+5. The following folder and file structure will be generated by the archetype:
+
+    ```plain
+    ~/src/
+        |--- acme-store/
+              |--- all/
+              |--- core/
+              |--- samplecontent/
+              |--- ui.apps/
+              |--- ui.content/
+              |--- README.md
+              |--- pom.xml
+    ```
+
+    * **Parent POM** - deploys maven modules and manages dependency versions
+    * **core** - Java bundle containing all core functionality like Sling Models, OSGi services, and other component related Java code such as servlets or request filters.
+    * **ui.apps** - contains the `/apps` part of the project, including proxy components for CIF core components and WCM Core Components.
+    * **ui.content** - contains structural content and configurations (`/content`, `/conf`) including CIF templates like catalog template, category template, and search result template.
+    * **samplecontent** - contains some sample assets and pages to get started. Once a project reaches maturity, this module can be removed and should be removed prior to production.
+    * **all** - combines all of the sub-modules and CIF dependencies into a single package that can be installed to a local AEM instance.
+
+## Deploy the project to AEM
```

Comment:
I think we should also mention the dispatcher setup here. Otherwise client-side components like the cart will not work. See https://github.com/adobe/aem-core-cif-components/tree/master/dispatcher.

Reply:
@herzog31 I think eventually we will have a separate guide dedicated to dispatcher setup. Should we call this out in maybe a "Next Steps" section?